src/rdata.cpp

```diff
@@ -444,6 +445,21 @@ void ReturnData::handleSx0Backward(const Model &model,
     auto xQBpreeq = preeq.getAdjointQuadrature();
     for (int ip = 0; ip < model.nplist(); ++ip)
         xQB[ip] += xQBpreeq[ip];
+
+    /* Prefer explicit type declaration here, since this MUST be a reference
+     * (for memory reasons), rather than passing this thing by value */
+    const AmiVectorArray &sx0preeq = preeq.getStateSensitivity();
+    auto xBpreeq = preeq.getAdjointState();
```

Comment:
```suggestion
    const auto& xBpreeq = preeq.getAdjointState();
```
(or is there a reason for copying?)